frontend/components/SeasonalCover.vue

```diff
@@ -0,0 +1,57 @@
+<script setup lang="ts">
+import springURL from "@/assets/images/covers/coverimage-spring.jpg";
+import summerURL from "@/assets/images/covers/coverimage-summer.jpg";
+import autumnURL from "@/assets/images/covers/coverimage-autumn.jpg";
+import winterURL from "@/assets/images/covers/coverimage-winter.jpg";
+const seasons = {
+  spring: {
+    image: springURL,
+    author: "Dick Boetekees",
+    // seasons.classes controls how the image is fitted inside the container; every image has its own 'best' fit
+    classes: "align-items-end",
+  },
+  summer: {
+    image: summerURL,
+    author: "Bert Spiertz",
+    classes: "align-items-middle",
+  },
+  autumn: {
+    image: autumnURL,
+    author: "Ivar Pel",
+    classes: "align-items-middle",
+  },
+  winter: {
+    image: winterURL,
+    author: "Simona Evstatieva",
+    classes: "align-items-middle",
+  },
+};
+//only considers Dutch seasons, not Australian
+function getSeasonalCoverImage() {
+  let month = new Date().getMonth() + 1; //const month goes from 1 to 12
```

Comment:
I'd use `const` instead of `let` here. In general, you should try to use `const` whenever you can and use `let` if you really need a mutable variable.

I'd also remove the comment at the end of the line. It's clear from the rest of the code what you're doing here.

```suggestion
  const month = new Date().getMonth() + 1;
```